packages/devtools_app/lib/src/shared/analytics/prompt.dart

```diff
@@ -89,34 +96,9 @@ class _AnalyticsPromptState extends State<AnalyticsPrompt>
   }
 
   Widget _analyticsDescription(TextTheme textTheme) {
-    return RichText(
-      text: TextSpan(
-        children: [
-          TextSpan(
-            text: 'DevTools reports feature usage statistics and basic '
-                'crash reports to Google in order to help Google improve '
-                'the tool over time. See Google\'s ',
-            style: textTheme.titleMedium,
-          ),
-          TextSpan(
-            text: 'privacy policy',
-            style:
-                textTheme.titleMedium?.copyWith(color: const Color(0xFF54C1EF)),
-            recognizer: TapGestureRecognizer()
-              ..onTap = () {
-                unawaited(
-                  launchUrl(
-                    'https://www.google.com/intl/en/policies/privacy',
-                  ),
-                );
-              },
-          ),
-          TextSpan(
-            text: '.',
-            style: textTheme.titleMedium,
-          ),
-        ],
-      ),
+    return SelectableText(
+      controller.consentMessage!,
```

Comment:
looks like now we won't be able to have a link to show google's privacy policy. Before we had a link text span for 'privacy policy'. Is this okay?

Reply:
Yeah we don't have the hyperlink anymore because the text is now getting fetched, but I did wrap the text with the `SelectableText` widget so that users can copy and paste the url still if they wanted to read about the policy.

Is that good enough?

Reply:
Does the hyperlink come as part of the message? What do you think about using a RegExp to pull the link out of the text and put it in a link text span? I think this would be relatively easy to do.

Reply:
yeah we can do that, i can add that in

Reply:
<img width="791" alt="image" src="https://github.com/flutter/devtools/assets/42216813/9fd9d1be-757b-4960-a863-1735c2bc6ce6">

I was able to swap out the text and user a hyperlink